_includes/header.html

```diff
@@ -14,6 +14,9 @@
         </li>
       {% endif %}
     {% endfor %}
+    <li class="nav-item">
+      <a href="https://euruko2017.org/2017/06/21/diversity-program.html">Diversity</a>
```

Comment:
could you use absolute paths without the protocol and domain? I think that's the convention we are using in the project. It would also work locally